test/system/700-play.bats

```diff
@@ -981,3 +981,45 @@ _EOF
     run_podman pod rm -t 0 -f test_pod
     run_podman rmi -f userimage:latest $from_image
 }
+
+@test "podman play with automount volume" {
+    cat >$PODMAN_TMPDIR/Containerfile <<EOF
+FROM $IMAGE
+RUN sh -c 'mkdir /test1 /test2'
+RUN sh -c 'touch /test1/a /test1/b /test1/c'
+RUN sh -c 'touch /test2/asdf /test2/ejgre /test2/lteghe'
+VOLUME /test1
+VOLUME /test2
+EOF
+
+    run_podman build -t automount_test -f $PODMAN_TMPDIR/Containerfile
+
+    fname="/tmp/play_kube_wait_$(random_string 6).yaml"
+    echo "
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test
+  name: test_pod
+spec:
+  restartPolicy: Never
+  containers:
+    - name: test1
```

Comment:
ARGH. As a bear of very little brain, I found it impossible to figure out "is this test1 the volume or test1 the container?" Could you please rename this to `testctr` (or whatever), and similarly update lines 1014, 1018, 1023?